src/lib/test-handler.js

```diff
@@ -69,37 +69,30 @@ export default class TestHandler {
   createWorker(scenario) {
     this.verboseLogger.log('Initializing worker for: ' + scenario.featureFile + ':' + scenario.scenarioLine);
 
-    let workerModulePath = path.join(__dirname, 'worker');
-    let workerEnv = _.merge(
-      process.env,
-      this.options.workerEnvVars,
-      scenario,
-      {testOptions: JSON.stringify(this.options)}
-    );
-    let worker = fork(workerModulePath, [], {
-      env: workerEnv,
-      silent: !this.options.inlineStream
-    });
+    let cucumberPath = './node_modules/.bin/cucumber-js';
```

Comment:
i don't think this actually works. cucumber.js in the module root isn't actually the module's CLI entry point.